mkdocs.yaml

```diff
@@ -54,29 +54,38 @@ plugins:
         - ^(?!(.*/)?assets/).*\.(?!(.*\.)?md|(.*\.)?svg|(.*\.)?png|(.*\.)?gif|(.*\.)?jpg).*$
         - ^(.*/)?[^.]*$
   - macros
+  - mkdocs-video
   - same-dir
   - search
 
 markdown_extensions:
+  - abbr
   - admonition
   - attr_list
   - codehilite:
       guess_lang: false
+  - pymdownx.emoji:
+      emoji_index: !!python/name:materialx.emoji.twemoji
+      emoji_generator: !!python/name:materialx.emoji.to_svg
   - fontawesome_markdown
   - footnotes
-  - md_in_html
```

Comment:
I'll submit a pull request to `autoware-documentation`.

Reply:
Thanks :bow:

Reply:
https://github.com/autowarefoundation/autoware-documentation/pull/362